neqo-transport/src/connection/mod.rs

```diff
@@ -2765,7 +2765,11 @@ impl Connection {
                 initial_sent = Some(sent);
                 needs_padding = true;
             } else {
-                if pt == packet::Type::Handshake && self.role == Role::Client {
+                if pt == packet::Type::Handshake
```

Comment:
I realize, looking at this, that we might want this instead.


```suggestion
                if pt.is_long()
```

Reply:
For clarity, this would be a problem with 0-RTT as well.